help/setting-up-zulip-for-a-class.md

```diff
@@ -172,7 +172,7 @@ how to assign roles and permissions for a class.
 - Set [who can create and manage user groups][user-group-permissions].
   (Recommended: Admins and moderators)
 
-[user-group-permissions]: /help/manage-user-groups#configure-who-can-create-and-manage-user-groups
+[user-group-permissions]: /help/manage-user-groups#configure-who-can-create-user-groups
```

Comment:
Since there is no single section that encompasses both create and manage, I've pointed this link to `create` for now.